README.md

```diff
@@ -1475,7 +1475,7 @@ of usage.
 
 ## Release notes
 
-See information about breaking changes and release notes [here](https://github.com/pleerock/routing-controllers/tree/master/doc/release-notes.md).
+See information about breaking changes and release notes [here](https://github.com/typestack/routing-controllers/blob/master/CHANGELOG.md).

```

Comment:
Thanks for spotting this! We should use a relative URL: `[here](CHANGELOG.md)`Please update your PR and I will merge it.

Reply:
Done